src/core/Quote.ml

```diff
@@ -104,6 +104,10 @@ let rec quote_con (tp : D.tp) con =
     and+ tsnd = quote_con fib snd in
     S.Pair (tfst, tsnd)
 
+  | D.Signature sign, _ ->
+     let+ tfields = quote_fields sign con in
```

Comment:
btw this seems to have some strange indentation

Reply:
Fixed!